documentation/assemblies/assembly-kafka-bridge-quickstart.adoc

```diff
@@ -21,7 +21,7 @@ Use this quickstart to try out the Strimzi Kafka Bridge in your local developmen
 * Create a Kafka Bridge consumer
 * Perform basic consumer operations, such as subscribing the consumer to topics and retrieving the messages that you produced
 
-In this quickstart, HTTP requests are formatted as curl commands that you can copy and paste to your terminal. Access to a Kubernetes cluster is required; to run and manage a local Kubernetes cluster, use a tool such as Minikube, CodeReady Containers, or MiniShift.
```

Comment:
I don't think we need this, as we mention local or remote Kubernetes cluster in prereqs. It was showing downstream too